contents/docs/experiments/frequentist-method.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Frequentist statistical method
+---
+
+> You can set the default statistical method for experiments in **Settings > Organization > General**. You can also override this on a per-experiment basis in each experiment's settings.
+
+PostHog supports two statistical approaches: **Bayesian** and **Frequentist**. This page explains how our **Frequentist** engine works, what metrics it supports, and how to interpret its results.
+
+Frequentist testing is the classical statistics approach used in most academic and industry-standard A/B testing tools. Instead of assigning probabilities to outcomes, it asks: _“If there were no real difference between variants, how likely would we be to observe results like this?”_ That’s the foundation for the p-value and confidence interval you'll see in experiment results.
+
+## Supported metric types
+
+We support two metric types for Frequentist analysis:
+
+### Funnel
+
+Calculates the percentage of users for whom the metric occurred at least once (e.g. "Signed up", "Clicked CTA"). These are binary outcomes: the event either happened for a user or it didn’t.
+
+### Mean
+
+Tracks the value of the metric per user (e.g. revenue, session duration, number of clicks). These are continuous values, and typically include more variation per user.
+
+## How the Frequentist engine works
+> To verify how the statistical calculations are implemented, [see the code in our GitHub repo](https://github.com/PostHog/posthog/tree/master/products/experiments/stats/frequentist).
+
+We use a two-sided **Welch’s t-test**, a statistical method that works well even when sample sizes and variances differ between the test and control groups.
+
+Here's how the analysis works behind the scenes:
+
+### Step 1: Estimate the effect
+
+We start by calculating the difference between the test and control groups. Depending on your experiment configuration, this can be an **absolute** difference (e.g. +0.3 conversions per user) or a **relative** difference (e.g. +25% increase compared to control).
```

Comment:
nit: control before test, to be consistent
```suggestion
We start by calculating the difference between the control and test groups. Depending on your experiment configuration, this can be an **absolute** difference (e.g. +0.3 conversions per user) or a **relative** difference (e.g. +25% increase compared to control).
```